awx/ui_next/src/components/LoadingSpinner/LoadingSpinner.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Spinner as PFSpinner } from '@patternfly/react-core';
+import styled from 'styled-components';
+
+const Spinner = styled(PFSpinner)`
+  position: fixed;
+  top: 50%;
+  left: 50%;
```

Comment:
This doesn't exactly center the element; it puts the top left corner in the center. You probably want something more like

```
top: calc(50% - 28px);
left: calc(50% - 28px);
```
(28px is about half the spinner's size)